server/views/pages/create/checkAnswers.njk

```diff
@@ -130,7 +130,7 @@
                         
                     {% endif %}
                     
-                    <a href={{ backLink }} class="govuk-link govuk-!-font-size-19">Return to caselist</a>
+                    <a href="{{ backLink }}" class="govuk-link govuk-!-font-size-19">Return to caselist</a>
```

Comment:
when no backlink was present this was generating a url like: `/licence/create/id/123/class=%22govuk-link` due to lack of quotes